### PR TITLE
Isolate JCenter Dependencies

### PR DESCRIPTION
**Pull request by JcMinarro**
### Description
Isolate JCenter Dependencies
It will help us to complete https://stream-io.atlassian.net/browse/CAS-666
Majority of the dependencies already have an opened issue

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- ~[ ] Changelog updated with client-facing changes~
- ~[ ] New code is covered by unit tests~
- ~[ ] Comparison screenshots added for visual changes~
- [x] Reviewers added

### CODE REVIEW

**Review by leandroBorgesFerreira — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -36,12 +42,46 @@ allprojects {
...
+
+                // Remove when https://github.com/facebook/flipper/issues/1775 is closed
+                includeGroup("com.facebook.flipper")
+
+                // Remove when https://github.com/Kotlin/dokka/issues/41 is closed
+                includeGroup("org.jetbrains.dokka")
+
+                includeModule("net.yslibrary.keyboardvisibilityevent", "keyboardvisibilityevent")
+                includeModule("me.relex", "photodraweeview")
+                includeModule("org.jetbrains.trove4j", "trove4j")
```
Comment (zsmb13) [inline build.gradle:60]:
```suggestion
                // https://youtrack.jetbrains.com/issue/IDEA-261387
                includeModule("org.jetbrains.trove4j", "trove4j")
```

File: build.gradle [hunk 2]
```diff
@@ -36,12 +42,46 @@ allprojects {
...
+                // Remove when https://github.com/Kotlin/kotlinx.html/issues/81 is closed
+                includeModule("org.jetbrains.kotlinx", "kotlinx-html-jvm")
+
+                //Remove when https://github.com/JetBrains/markdown/issues/64 is closed
+                includeModule("org.jetbrains", "markdown")
+
+                // Remove when https://github.com/MarkusAmshove/Kluent/issues/190 is closed
+                includeGroup("org.amshove.kluent")
+
+                includeGroup("androidx.databinding")
```
Comment (zsmb13) [inline build.gradle:74]: Isn't this in `google()`? https://maven.google.com/web/index.html?q=databinding

File: build.gradle [hunk 3]
```diff
@@ -36,12 +42,46 @@ allprojects {
...
+            content {
+
+                // Remove when https://github.com/google/ExoPlayer/issues/5246 is closed
+                includeGroup("com.google.android.exoplayer")
+
+                // Remove when https://github.com/facebook/flipper/issues/1775 is closed
+                includeGroup("com.facebook.flipper")
+
+                // Remove when https://github.com/Kotlin/dokka/issues/41 is closed
+                includeGroup("org.jetbrains.dokka")
```
Comment (zsmb13) [inline build.gradle:56]: Isn't it enough to list Dokka above, in the buildscript repositories section?

Comment (JcMinarro) [reply build.gradle:56]: It is needed because the plugin is added to all modules, not only to the main one

File: build.gradle (hunk 2 above)
Comment (JcMinarro) [reply build.gradle:74]: Yep, it is, I don't know why I had some issues yesterday with this dependency

**Pull request merged**